packages/form-core/src/FormControlMixin.js

```diff
@@ -812,6 +813,20 @@ const FormControlMixinImplementation = superclass =>
       );
     }
 
+    /**
+     * TODO: Extend this in choice group so that target is always a choice input and multipleChoice exists.
```

Comment:
I considered this in the beginning and decided to keep all logic in one place for readability.

If we only have one condition method to override in choiceGroup (like this) then it would be acceptable.

Could we maybe give this method a 'positive name' (like `_repropagationCondition`)?